docs/content/reference/upgrade_dependencies.html.markdown

```diff
@@ -0,0 +1,18 @@
+# Upgrading Dependencies
```

Comment:
There needs to be [front matter](https://gohugo.io/content-management/front-matter/) here. Additionally, see [Headings and titles](https://developers.google.com/style/headings) for guidance on writing headings.

Weight 65 will put this just above "Promote to GA"

```suggestion
---
title: Upgrade provider dependencies
weight: 65
---
# Upgrade provider dependencies
```

Reply:
done.